portfolio/src/main/webapp/script.js

```diff
@@ -14,18 +14,30 @@
 
 async function getFormMessage() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('quote-container').innerHTML = quote;
+  const jsonComments = await response.text();
+  const comments = JSON.parse(jsonComments);
+  let node;
```

Comment:
You don't need to keep reassigning this since you don't use it outside of the loop. Just make it a `const node = document.createElement('p');` within the loop.

Reply:
done